src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -83,11 +83,14 @@ function BoxAISidebarContent(props: ApiWrapperWithInjectedProps) {
         setCacheValue('agents', { agents, requestState, selectedAgent });
     }
 
-    const handleUserIntentToUseAI = () => {
+    const handleUserIntentToUseAI = (userHasInteracted: boolean = false) => {
```

Comment:
[nit] I think `hasUserInteracted` would be more proper form

Reply:
That might be true, however I've named it the same way in content-answers, so for consistency I'd leave it like that.